Makefile

```diff
@@ -16,24 +14,29 @@ help:
 	${MAKE} build
 
 build:
-	-rm .docker-build
-	${DOCKERCOMPOSE} -f docker-compose.yml build irc
+	docker build -t standup_irc .
 	touch .docker-build
 
 run: .docker-build
-	${DOCKERCOMPOSE} up irc
+	docker run --env-file .env_dev -v "$PWD:/app" standup_irc
```

Comment:
This doesn't account for the postgres requirement. Though maybe we don't need a run rule at all?